SR2019/2019-05-28-single-item-kit-chase.md

```diff
@@ -0,0 +1,23 @@
+---
+to: Teams with outstanding single items of kit
+subject: Outstanding kit
+---
+
+Hello,
+
+According to our records, you still have some of our kit. As the competition has ended, we would like this back!
+
+At the end of the competition, we checked your kit in, and some items were missing. Below are the items we require back:
+
+## Missing items
+
+-
+-
+
+Please search anywhere you may have worked on your robot for these items. If you are unable to find any of these items, please [let us know](mailto:teams@studentrobotics.org).
+
+## Shipping items
+
+Please insure the kit during shipping and use a courier which supports tracking, to ensure the kit isn't lost.
+
+Please securely pack the kit and ship it to the address listed on [our website](https://studentrobotics.org/contact/). Once you have dispatched it, please [let us know](mailto:teams@studentrobotics.org). Please ensure your school name is written on the outside of the packaging.
```

Comment:
We should ask that they include their TLA on the outside of their packaging, so we know who sent what when it arrives.